src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -223,11 +240,19 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
     webviewPanel.onDidChangeViewState(
       () => {
         if (webviewPanel.visible) {
+          this._activeWebviewPanel = webviewPanel;
+          this._activeDocument = document;
           vscode.commands.executeCommand(
             "one.explorer.revealInOneExplorer",
             document.fileName
           );
         }
+        if (
+          !webviewPanel.visible &&
+          this._activeWebviewPanel === webviewPanel
+        ) {
+          this._activeWebviewPanel = undefined;
+        }
```

Comment:
Q) can you explain why this block is needed and what it does?

Reply:
> Q) can you explain why this block is needed and what it does?


It makes `_activeWebviewPanel` inactive in case it is invisible. If the user selects another view in oneexplorer, e. g. `CircleEditor' or just edits .cfg file in text mode, it's inactive. But   i may try to use `when' clause instead, like the following
```
 "when": "CfgEditor.active"
```
in `package.json` file

Reply:
> i may try to use when' clause instead, like the following ...

IIUC, `ctrl+D` won't do anything when this View is hidden.
If it works, this block can be removed in this PR?

Reply:
> IIUC, `ctrl+D` won't do anything when this View is hidden.

Ahh... Ok. I see. Thank you.

> If it works, this block can be removed in this PR?

Yes. It will be removed.